lib/registrant.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+class Registrant
+
+  attr_reader :name,
+              :age,
+              :permit,
+              :license_data
+
+  attr_accessor :permit, :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {:written=>false, :license=>false, :renewed=>false}
+  end
+
+  def permit?
+    if @permit
+      true
+    else
+      false
+    end
+  end
+
+  def earn_permit
+    @permit = true
+  end
+end
```

Comment:
You don't need line 7 because we never need to call `.permit` on a Registrant object. In your `permit?` method, you could simplify this a bit to look like: 

```ruby 
def permit? 
   @permit 
end 
```

since `@permit` will automatically return true or false.